wordpress-mobile/WordPress-iOS
# Fix ghost not disappearing from Insights cards in certain conditions

Fixes #17904

To test:
Pre-requisite: use a wp.com account with one or more site that have stats, preferably with many accesses

Testing:
1. Build/run and go to "My Site"
2. Tap on "Stats" then "Insights" (if you're not already there)
3. Screen should load normally
4. Navigate / back out to "My Site"
5. Repeat step 2
6. No ghost cards should appear

## Regression Notes
1. Potential unintended areas of impact
Stats -> Insights

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual test and run UI tests

3. What automated tests I added (or what prevented me from doing so)4. 
Some changes to the UI tests
PR submission checklist:
